ct: Add dark mode support to coverage pages

jchristgit opened the pull request:
When common test is tasked with generating coverage via `-cover`, invert the colours of the webpage when the user prefers a dark color scheme. Similar support was added to the other CT-generated pages in commit bbe14c9ff74d7f87d74622f7918e6a8d53963a4d.

## Screenshots

![image](https://github.com/erlang/otp/assets/22679924/9c6e10d5-e47f-4576-a0eb-c4b989439831)

![image](https://github.com/erlang/otp/assets/22679924/3279da3a-a5ea-404e-903c-f6e2e8338e40)

u3s commented:
have you checked Edge browser? can you check it?
my instance renders background as white ...

![image](https://github.com/erlang/otp/assets/4620572/6b4b2a5b-c389-4ce1-8d0c-6b3e29ab6624)

jchristgit commented:
Sorry for the delay with getting back to you on my PRs, life has been
very busy lately. I will try to test this with Edge and investigate this
by next week. I hope you have an enjoyable Easter weekend :-)

u3s commented:
> Sorry for the delay with getting back to you on my PRs, life has been very busy lately. I will try to test this with Edge and investigate this by next week. I hope you have an enjoyable Easter weekend :-)

great thanks!

jchristgit commented:
I rebased from master and reproduced the issue on Edge.

The issue seems to be that Edge needs to have an explicitly set
background color to be able to rotate it. The new changes work for me on
Firefox and Edge.

Whilst checking the PR I also noticed the individual coverage pages
didn't include any dark theme, so I'm wondering whether I forgot to push
something there. I've added a snippet I believe to work, but
unfortunately after rebasing from main I now get a segfault when
compiling, even after cleaning :( I think it's due to my (very wonky)
setup right now, so I'm going to convert this to a draft and get back to
you when I have access to my proper dev machine again.

u3s commented on lib/tools/priv/styles.css:
```diff
@@ -94,3 +94,17 @@ td.source {
   white-space: pre;
   font: 12px monospace;
 }
+
```
is adjustment in this file needed for this PR functionality?

u3s commented:
> @u3s I got an e-mail about a comment you made on the tools CSS but I think you deleted it, did the question solve itself or should I change something?

no. I think I've made it to quickly and answered it to myself later.
thanks for contributions! sorry for delays.
